src/util/logs.rs

```diff
@@ -0,0 +1,64 @@
+use crate::subscriptions;
+use colored::Colorize;
+
+pub fn format_attr_log(mut log: subscriptions::deployment_logs::LogFields) {
+    if !log.attributes.is_empty() {
+        let mut timestamp: Option<String> = None;
+        let mut level: Option<String> = None;
+        let message = log.message;
+        let mut others = Vec::new();
+        // for some reason, not all have "" around the value
+        for attr in &mut log.attributes {
+            if !attr.value.starts_with('"') {
+                attr.value.insert(0, '"');
+            };
+            if !attr.value.ends_with('"') {
+                attr.value.push('"');
+            }
+        }
+        // get attributes using a match
+        for attr in &log.attributes {
+            match attr.key.to_lowercase().as_str() {
+                "timestamp" | "ts" | "time" => {
+                    timestamp = Some(attr.value.clone().replace('"', ""))
+                }
+                "level" | "lvl" => level = Some(attr.value.clone()),
+                _ => others.push(format!(
```

Comment:
There might be nested attributes in here too. I feel like we should probably just throw these away for now? Or perhaps cap attributes to a certain length?

Here's an example response with some nested stuff.

![CleanShot 2023-11-17 at 14 16 12@2x](https://github.com/railwayapp/cli/assets/587576/1c2028f7-1b49-4ad0-a91e-65b80ee9d554)